packages/core/src/configuration.ts

```diff
@@ -182,6 +183,21 @@ const scatterChart: ScatterChartOptions = Tools.merge({}, axisChart, {
 	}
 } as ScatterChartOptions);
 
+/**
+ * options specific to bubble charts
+ */
+const bubbleChart: BubbleChartOptions = Tools.merge({}, scatterChart, {
+	bubble: {
+		radiusRange: (chartSize, data) => {
```

Comment:
This is to constrain the scale of the radii? Definitly going to need some docs around this